### PR TITLE
feat: adds preview component

### PR DESCRIPTION
**Pull request by eljefe223**
# Description
Stubs in preview component into marketing site.

Screen Shot 2020-05-11 at 8 17 01 AM

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [x] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: sites/fast-website/src/app/components/fast-frame/fast-frame.styles.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    .content {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        align-items: start;
+    }
+
+    .preview {
+        display: flex;
+        background: #c8c8c8;
```
Comment (nicholasrice) [inline sites/fast-website/src/app/components/fast-frame/fast-frame.styles.ts:32]: Should this color be calculated?

Comment (eljefe223) [reply sites/fast-website/src/app/components/fast-frame/fast-frame.styles.ts:32]: This is just temporary to show the bounds of the preview container.

File: sites/fast-website/src/app/components/fast-frame/fast-frame.template.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                            fill="#C8C8C8"
```
Comment (nicholasrice) [inline sites/fast-website/src/app/components/fast-frame/fast-frame.template.ts:34]: Should the fills be colored with CSS?

File: sites/fast-website/src/app/components/fast-frame/fast-frame.template.ts [hunk 3]
```diff
@@ -0,0 +1,94 @@
+import { html } from "@microsoft/fast-element";
+import { FastFrame } from "./fast-frame";
+
+export const FastFrameTemplate = html<FastFrame>`
+    <template>
+        <div class="wrapper">
+            <fast-tabs orientation="vertical" id="myTab" activeId="TabTwo">
+                <fast-tab id="TabOne" title="Mode">
+                    <svg
```
Comment (radium-v) [inline sites/fast-website/src/app/components/fast-frame/fast-frame.template.ts:9]: I'll add an svg loader so we can keep these icons isolated.

File: sites/fast-website/src/app/components/fast-frame/fast-frame.template.ts (hunk 2 above)
Comment (eljefe223) [reply sites/fast-website/src/app/components/fast-frame/fast-frame.template.ts:34]: Yes, But I'm holding off on styling overrides on our controls. We want to do the work on separating out the styling from the web components first before we spend a bunch of time overriding values.

**Pull request merged**